spec/models/solr_document_spec.rb

```diff
@@ -4,21 +4,24 @@
   describe '#object_title' do
     subject(:title) { solr_doc.object_title }
     let(:solr_doc) { described_class.new(values) }
+    let(:user) { create(:archivist1) }
 
     before do
       # Don't index workflow stuff
       allow_any_instance_of(Dor::ProcessableIndexer).to receive(:to_solr).and_return({})
     end
 
     context 'for a new item' do
-      let(:values) { Hydrus::Item.new.to_solr }
+      let(:values) { Hydrus::Item.create.to_solr }
```

Comment:
This change and the one on line 23 are the only ones I don't fully understand. The problem? Calling `Hydrus::Item.new.to_solr` winds up trying to indexing workflows, but there's no druid yet, so the call to index workflows generates a 404 because there's no `objects//workflows` route (the druid should be between the slashes but there's no druid). Is this kosher or should I attempt a different fix? @sul-dlss/infrastructure-team

Reply:
I think this is fine.